docs/quickstart/php.md

```diff
@@ -101,7 +101,7 @@ The simplest way to do this is to download pre-compiled binaries for your platfo
 To compile the gRPC PHP Protoc Plugin:
 
 ```sh
-$ git clone https://github.com/grpc/grpc
+$ git clone -b {{ site.data.config.grpc_release_branch }} https://github.com/grpc/grpc
```

Comment:
How about

```sh
git clone --recursive -b {{ site.data.config.grpc_release_branch }} https://github.com/grpc/grpc
```

instead of a clone and a subsequent submodule init?